README.rst

```diff
@@ -100,6 +100,8 @@ pre-release testers) we cannot guarantee that every release will run
 on all non-Linux systems. Please report any bugs you find, and we will
 try to fix them.
 
+You can also run S3QL in a Docker container. See <insert docker README link
```

Comment:
I'd be curious under which situations someone would want to do that. Given that the fuse module still has to be loaded in the host system kernel and that special privileges are needed for the guest, security and isolation are probably not applicable here. For dependency management, Python's venv system would probably also fit the bill better. So maybe add a sentence describing the conditions under which this makes sense?

Reply:
In most cases the fuse module is already loaded on the host by default. Indeed security is not at all a concern here, but rather, as you hint with the mention of venvs, it's just easier to deal with in practice (imo)?

Even with venvs it's not all that nice to do things directly on the host.
Like, atm, fedora 33 defaults to python 3.9 as provider of python3, which doesn't successfully build and run the project, and that's the kind of thing that having a "whatever we baked in this image, it will run good" is quite convenient for. I don't want system updates to suddenly break my apps anymore (or make the choice to never update and run debian oldstable till the end of times :slightly_smiling_face: )

If you disagree with all of these, lastly, in my case, it's that you must use containerised apps with k8s anyway, there isn't a "run on host directly" way that isn't awfully hacky

---

however I agree that this should be documented, either in the main README or the one in contrib/docker. Whichever one you think it's more appropriate to put this in!

Reply:
I think it's best to have 1-3 sentences in README. If you want, you can go into more detail in the Docker readme.